packages/bento-design-system/src/util/PaddingConfig.ts

```diff
@@ -0,0 +1,13 @@
+import { BentoSprinkles } from "../internal";
+
+type PaddingKey =
+  | "padding"
+  | "paddingX"
+  | "paddingY"
+  | "paddingLeft"
+  | "paddingRight"
+  | "paddingTop"
+  | "paddingBottom";
+type Padding = 0 | BentoSprinkles["padding"];
+
+export type PaddingConfig = { [k in PaddingKey]?: Padding };
```

Comment:
Wouldn't be better to use a more precise union type here, as we did for `BorderRadiusConfig`?
Just to avoid cases like `{ paddingX: 8, paddingLeft: 4, paddingRight: 4 }`.

Also, being able to set `defaultPadding: 4` instead of `defaultPadding: { padding: 4 }` feels more natural to me.

Reply:
Didn't understand the first point. Would should it be breaking? Isn't it an additional config entry?
Not sure about the second point either... it's true for sure, but I would prefer to have consistency between configs and avoid nonsense configs instead of optimizing something you have to write only once per project.